elegantthemes/d5-dev-tool
# Transitioned from "D5 Modal Dev State Monitor" to "Divi 5 Dev Tool" Name

### Summary
This PR includes all the necessary renaming to transition to the "Divi 5 Dev Tool" name. The change ensures the name is meaningful and reflective of the extension's functionalities.

### How To Test
1. Run a successful `npm install`
2. Run a successful `npm run start` 
3. The extension should be loaded and functional on D5 visual builder.
